src/components/Editor/ConditionalPanel.js

```diff
@@ -22,6 +24,10 @@ function renderConditionalPanel({ condition, closePanel, setBreakpoint }:
   ReactDOM.render(
     dom.div(
       { className: "conditional-breakpoint-panel" },
+      dom.div({
+        className: "prompt",
+        dangerouslySetInnerHTML: { __html: "&raquo;" }
```

Comment:
Spoke to @jasonLaster and we could likely use the unicode character itself rather than `dangerouslySetInnerHTML`. But that's a minor change. 😄